### ISSUE TITLE
Error: endpoint not found 

### ISSUE THREAD

**Issue by naimo84** [hobbyquaker/node-red-contrib-zigbee#34]
Hello everybody,

I have a CC2531. This works perfectly with my HUE bulbs. I can control everything. That is not the problem. 
But I keep getting the following message: endpoint not found 0x001788010275aec2 11
Although the bulbs are controllable, the status is "offline".
It is also noticeable that the file /data/zigbee/e13a1ef9.f7921/dev.db grows larger with each Node-Red reload. The same number of devices, just a new JoinTime.

I don't see my mistake ;-)

Can you help me with this? Do you need more information?

Thanks und regards,
Benni

--------------------

![2019-05-13 07_37_59-Clipboard](https://user-images.githubusercontent.com/11535502/57598152-5c6e2080-7552-11e9-918a-781a5bb85052.png)

---------------------

![2019-05-13 07_40_48-Node-RED _ node neumann space - Brave](https://user-images.githubusercontent.com/11535502/57598179-7c9ddf80-7552-11e9-93f6-0d4d5562efc8.png)

----------------------
Here is some log:

```
13 May 05:11:54 - [debug] [zigbee-shepherd:zigbee shepherd] stopping
13 May 05:11:54 - [debug] [zigbee-shepherd:zigbee shepherd] stopped shepherd
13 May 05:11:54 - [info] [zigbee-shepherd:zigbee shepherd] persistPath /data/zigbee/e13a1ef9.f7921
13 May 05:11:54 - [info] [zigbee-shepherd:zigbee shepherd] connecting /dev/ttyACM0 {"baudRate":115200,"rtscts":true}
13 May 05:11:54 - [debug] [zigbee-converter:ControlHueLight] topic=zigbee/0x001788010275aec2/set name=Wohnzimmer Schrank ieeeAddr=0x001788010275aec2 attribute=undefined payload={"brightness":0,"transition":3}
13 May 05:11:54 - [debug] [zigbee-converter:ControlHueLight] payload {"brightness":0,"transition":3}
13 May 05:11:54 - [error] [zigbee-shepherd:zigbee shepherd] endpoint not found 0x001788010275aec2 11
13 May 05:11:54 - [warn] [zigbee-converter:ControlHueLight] no conversion for transition
13 May 05:11:54 - [debug] [zigbee-converter:ControlHueLight] topic=zigbee/0x00178801011cb05a/set name=Wohnzimmer Couch ieeeAddr=0x00178801011cb05a attribute=undefined payload={"brightness":0,"transition":3}
13 May 05:11:54 - [debug] [zigbee-converter:ControlHueLight] payload {"brightness":0,"transition":3}
13 May 05:11:54 - [error] [zigbee-shepherd:zigbee shepherd] endpoint not found 0x00178801011cb05a 11
13 May 05:11:54 - [warn] [zigbee-converter:ControlHueLight] no conversion for transition
13 May 05:11:55 - [error] [zigbee-shepherd:zigbee shepherd] endpoint not found 0x001788010275aec2 11
13 May 05:11:55 - [error] [zigbee-shepherd:zigbee shepherd] endpoint not found 0x00178801011cb05a 11
13 May 05:11:56 - [info] [zigbee-shepherd:zigbee shepherd] coordinator 0x00124b0018e1b1c4 firmware version: 2.6.3 20190223
13 May 05:11:56 - [info] [zigbee-shepherd:zigbee shepherd] started panId: 0xbe9b channel: 11 (11)
13 May 05:11:56 - [info] [zigbee-shepherd:zigbee shepherd] ready
13 May 05:11:56 - [debug] [zigbee-shepherd:zigbee shepherd] device list unchanged
13 May 05:11:56 - [info] [zigbee-shepherd:zigbee shepherd] Currently 2 devices are joined:
13 May 05:11:56 - [info] [zigbee-shepherd:zigbee shepherd] 0x001788010275aec2 Wohnzimmer Schrank (Router Philips LCT010)
13 May 05:11:56 - [info] [zigbee-shepherd:zigbee shepherd] 0x00178801011cb05a Wohnzimmer Couch (Router Philips LST002)
13 May 05:12:12 - [info] [zigbee-shepherd:zigbee shepherd] permitJoin 0 all
13 May 05:12:14 - [info] [zigbee-shepherd:zigbee shepherd] permitJoin 0 all
13 May 05:12:17 - [info] [zigbee-shepherd:zigbee shepherd] permitJoin 0 all
13 May 05:12:35 - [info] [zigbee-shepherd:zigbee shepherd] permitJoin 0 all


```

**Comment by hobbyquaker** [comment]
> But I keep getting the following message: `endpoint not found 0x001788010275aec2 11`

have to investigate on this, no clue atm.

> Although the bulbs are controllable, the status is "offline".

should be fixed in 0.7.2

> It is also noticeable that the file /data/zigbee/e13a1ef9.f7921/dev.db grows larger

That's normal behavior of the zigbee-shepherd i think. will check

**Comment by naimo84** [comment]
- thanks for investigating :)

- the status is unfortunately still "offline" with version 0.7.2
